src/P5Functions/touchmouse.js

```diff
@@ -315,6 +315,8 @@ module.exports = function withTouch(p, curElement, attachEventHandler, document,
 
     if (delta && typeof p.mouseScrolled === 'function') {
       p.mouseScrolled();
+      e.preventDefault();
+      e.stopPropagation();
```

Comment:
Those 2 "prevent" statements should be outside of that condition. They should activate regardless.
Of course you coulda simply `return false;` too.  :P